data/mil2525d/utilities/military-features-utilities/template-gdb-toolbox/scripts/CreateTemplateGDB/UpdateDomains.py

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------------------
```

Comment:
Since this deviates from our current practice (standard or not - I can't tell without reading a lengthy doc, which likely pertains to packaging) of just putting things in the scripts folder - we should probably have a discussion before changing ( I'll let you and @mfunk have that discussion though)